napari/_qt/widgets/qt_dims_slider.py

```diff
@@ -553,8 +555,11 @@ def mouseReleaseEvent(self, event):
 
     def _on_click(self):
         """Toggle play/stop animation control."""
+        qt_dims = self.qt_dims_ref()
+        if not qt_dims:
```

Comment:
```suggestion
        if not qt_dims:  # pragma: no cover
```

To not generate noise in coverage reports